vehicle/ford/api.go

```diff
@@ -102,13 +102,19 @@ func (v *API) RefreshRequest(vin string) (string, error) {
 
 // WakeUp performs a wakeup request
 func (v *API) WakeUp(vin string) error {
-	uri := fmt.Sprintf("%s/api/dashboard/v1/users/vehicles?language=EN&skipRecall=true&country=USA&region=US&wakeupVin=%s", TokenURI, vin)
+	//	uri := fmt.Sprintf("%s/api/dashboard/v1/users/vehicles?language=EN&skipRecall=true&country=USA&region=US&wakeupVin=%s", TokenURI, vin)
+	/* uri := fmt.Sprintf("%s/api/dashboard/v1/users/vehicles?wakeupVin=%s", TokenURI, vin)
 
 	res, err := v.GetBody(uri)
 
 	if err == nil && res == nil {
 		err = errors.New("wakeup failed")
 	}
+	*/
+	r, err := v.RefreshRequest(vin)
+	if err == nil && r == "" {
+		err = errors.New("wakeup failed")
```

Comment:
refresh failed? das sollte dann aber in den v.RefreshRequest?

Reply:
hab ich rüber kopiert da der WakeUp API call nichts bringt aber der RefreshRequest weckt das Auto auf. Aber muss ich weiter testen.

Reply:
hab ich wieder zurück geändert - ich denke das ich einfach zu ungeduldig war.